root/static/scripts/edit/URLCleanup.js

```diff
@@ -1649,6 +1650,28 @@ const CLEANUPS = {
       return url;
     },
   },
+  'laboiteauxparoles': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?laboiteauxparoles\\.com', 'i')],
+    type: LINK_TYPES.lyrics,
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?(?:www\.)?laboiteauxparoles\.com\/(auteur|editeur|interprete|titre)\/([^\/?#]+)(?:.*)?$/, 'https://laboiteauxparoles.com/$1/$2');
+    },
+    validate: function (url, id) {
+      const m = /^https:\/\/laboiteauxparoles\.com\/(auteur|editeur|interprete|titre)\//.exec(url);
+      if (m) {
+        const prefix = m[1];
+        switch (id) {
+          case LINK_TYPES.lyrics.artist:
+            return {result: /^(?:auteur|interprete)$/.test(prefix)};
+          case LINK_TYPES.lyrics.label:
+            return {result: prefix === 'editeur'};
+          case LINK_TYPES.lyrics.work:
+            return {result: prefix === 'titre'};
+        }
+      }
+      return {result: false};
+    },
+  },
```

Comment:
This is not related to “Support laboiteauxparoles label links”. Please update ticket title/summary, PR description and (separate) commit message accordingly.

Reply:
Title fixed in https://github.com/metabrainz/musicbrainz-server/pull/1628/commits/fe06779e9d3e3d2a0ab1dac5e0fa8884a94bba00